etc/xdg/weston/weston.ini

```diff
@@ -2,6 +2,10 @@
 xwayland=true
 idle-time=0
 
+[output]
+name=eDP-1
+scale=2
```

Comment:
Is this still needed even with InputPlumber working now?

Reply:
Yes, without scaling the icons and window controls are really tiny and hard to press. Even with the 2x scaling it can be hard to use.